src/clj/game/cards/assets.clj

```diff
@@ -797,7 +805,7 @@
     :events {:corp-turn-begins
              {:effect (req (add-counter state side card :power -1)
                            (when (<= (get-in card [:counter :power]) 1)
-                             (system-msg state :corp "adds Public Support to his scored area and gains 1 agenda point")
+                             (system-msg state :corp "uses Public Support to add it to their score area as an agenda worth 1 agenda point")
```

Comment:
regarding his->their, how (in)consistent is it currently in code? Maybe it would be worth it doing a search & replace sweep?